about/moveit-status/index.markdown

```diff
@@ -5,15 +5,22 @@ date: 2013-12-08 05:22:09+00:00
 layout: page
 slug: moveit-status
 title: Status
-wordpress_id: 164
 ---
 
-This page highlights the status of MoveIt! components - it indicates how stable individual components are and how well they have been tested. It also indicates how much support users can expect for different components and robots. (See legend below for description of the codes.)
+# MoveIt! Status
 
+## Roadmap
 
-### Current STATUS (Components)
+Upcoming milestones as of September 2016:
 
+- [First release of Kinetic](https://github.com/ros-planning/moveit/issues/18)
+- [New release of Indigo](https://github.com/ros-planning/moveit/issues/100)
```

Comment:
Are you sure you want to have this section on the website?
This is pretty much transient information in my opinion.